frontend/src/pages/components/Navbar/AppsDropdownContent/Card.tsx

```diff
@@ -0,0 +1,75 @@
+import Image from "next/image";
+import clsx from "clsx";
+import { Solution } from "@/queries/navbar";
+
+const hoverEffect = clsx(
+  "hover:scale-[1.01] transform transition duration-100"
+);
+const cardBaseStyle = clsx(
+  "bg-background-2 rounded-lg border border-stroke text-wrap",
+  "p-4 flex gap-4 h-full lg:items-start"
+);
+const headerTextStyle = clsx("text-primary-purple font-bold text-base");
+const descriptionTextStyle = clsx(
+  "text-secondary-text text-base leading-tight"
+);
+
+interface CardProps {
+  solution: Solution;
+  variant: "large" | "medium" | "small";
+}
+
+const Card: React.FC<CardProps> = ({ solution, variant }) => {
+  return (
+    <a
+      key={solution?.solution_name}
+      href={solution?.url}
+      rel="noopener noreferrer"
+      target="_blank"
+      className={clsx(cardBaseStyle, hoverEffect, "flex-row", "w-full", {
+        "lg:flex-col lg:pb-8": variant === "large",
+        "lg:flex-row": variant === "medium" || variant === "small",
+        "lg:w-[380px]": true,
+      })}
+    >
+      <Image
+        src={solution?.logo_svg?.url}
+        alt={solution?.solution_name}
+        width={64}
+        height={64}
+      />
+      <div>
+        <h3 className={headerTextStyle}>{solution?.solution_name}</h3>
+
+        <h1
+          className={clsx(
+            headerTextStyle,
+            "text-primary-text leading-tight mt-1",
+            {
+              "text-base": true,
+              "font-normal": variant === "small",
+              "lg:text-[32px]": variant === "large",
+              "lg:text-[24px]": variant === "medium",
+              "lg:text-base": variant === "small",
```

Comment:
`text-base` is already in `headerTextStyle`.
I added text sizes to the tailwind config so that they comply with the design, you can use:
```
      base: ["1rem", "1.188rem"], // 16px
      lg: ["1.5rem", "1.813rem"], // 24px
      xl: ["2rem", "2.375rem"], // 32px
      '2xl': ["3rem", "3.625rem"], // 48px
      '3xl': ["4rem", "4.813rem"], // 64px
      '4xl': ["6rem", "7.188rem"], // 96px
```
```suggestion
              "font-normal": variant === "small",
              "lg:text-xl": variant === "large",
              "lg:text-lg": variant === "medium",
              "lg:text-base": variant === "small",
```